_topics/en/multisignature.md

```diff
@@ -0,0 +1,108 @@
+---
+title: Multisignature
+
+## Optional.  Shorter name to use for reference style links e.g., "foo"
+## will allow using the link [topic foo][].  Not case sensitive
+# shortname: foo
+
+## Optional.  An entry will be added to the topics index for each alias
+aliases:
+  - 2pECDSA
+  - Two-Party ECDSA (2pECDSA)
+
+## Required.  At least one category to which this topic belongs.  See
+## schema for options
+categories:
+  - Privacy Enhancements
+
+## Required.  Use Markdown formatting.  Only one paragraph.  No links allowed.
+## Should be less than 500 characters
+excerpt: >
+  **Multisignatures** are digital signatures that look like they were
+  created using only a single private key but which were actually
+  created using two or more keys.
+
+## Optional.  Use Markdown formatting.  Multiple paragraphs.  Links allowed.
+extended_summary: |
+  Only a single key and a single signature are published onchain when
+  multisignatures are used in a Bitcoin transaction, allowing an
+  unlimited number of signers to pay the same amount of transaction fee
+  that a single signer would pay for an otherwise identical transaction.
+  Multisignature payments being indistinguishable from single-signature
+  payments gives the creators of both types of payments greater privacy.
+
+  It's possible to create multisignatures for the ECDSA algorithm
+  supported by all versions of Bitcoin, although it's complex and
+  requires security assumptions beyond those normally required for
+  Bitcoin-style ECDSA signatures.  It's easier to create multisignatures
+  for [schnorr signatures][topic schnorr signatures], and several
+  algorithms for that are known, with [MuSig][topic musig] having been
+  specifically created for the needs of Bitcoin users.
+
+  **Terminology:** multisignatures should not be confused with
+  *multisig*, the use of public keys and signatures with Bitcoin's
+  `OP_CHECKMULTISIG` and `OP_CHECKMULTISIGVERIFY` opcodes (and the
+  `OP_CHECKSIGADD` opcode proposed for [tapscript][topic tapscript]).
+  The following table summarizes the differences between *multisignature*
+  and several related terms.
+
+  | Term | Keys | Messages<br>(e.g. tx inputs) | Signatures | Signers required | Notes |
+  |-|-|-|-|-|
+  | Multisig | `m` | `1` | `k` where `k<=m` | `k` | Uses `OP_CHECK*SIG` opcodes |
+  | Multisignature | `m` | `1` | 1 | `m` | Indistinguishable onchain from single-sig |
+  | Threshold signature | `m` | `1` | `1` | `k` where `k<=m` | Indistinguishable onchain from single-sig |
+  | Signature aggregation | `m` | `n` | 1 | `0` or `m`<br>(varies by scheme) | Proposed to allow a tx with multiple inputs to have only one signature, among other uses |
```

Comment:
I'd leave Signature aggregation out entirely for now. Perhaps add it if we later add a topic page for it.